### ISSUE TITLE
Switch of the handle text/flag

### ISSUE THREAD

**Issue by mh19971967** [IonDen/ion.rangeSlider#733]
Hi Denis,

First of all, congratulations on your very nice framework.

One short question:

**Is it possible to switch of the handle text/flag?**

Thanks very much and kind regards

**Comment by IonDen** [comment]
@mh19971967, You mean the label? Yes, add this to params: hide_from_to: true

**Comment by IonDen** [comment]
Yes, set `hide_from_to: true` and the from/to bubbles above the handles won't show at all.

```js
$(".js-range-slider").ionRangeSlider({
    type: "double",
    hide_from_to: true
});
```

Say so if that's not the element you meant.
